lua/neotest-go/color.lua

```diff
@@ -1,16 +1,136 @@
 local M = {}
 
--- Highlight some parts of the test output
-function M.highlight_output(output)
+local fmt = string.format
+local colors_patterns = require("neotest-go.patterns").colors
+
+local term = os.getenv("COLORTERM")
+local fullcolor = (term == "truecolor") or (term == "24bit")
+
+local function slice(tbl, first, last, step)
```

Comment:
is this different or better than using `vim.list_slice`?

Reply:
Its not